### ISSUE TITLE
Multiple Select - Svelte

### ISSUE THREAD

**Issue by apiotrowski312** [AgnosticUI/agnosticui#133]
**Describe the bug**

I am not able to use multiple Select as there is no way to get all selected options. I might be just doing something wrong, if that would be the case, sorry :( 

**To Reproduce**
1. Create Select with `isMultiple`
```
<Select
	multipleSize={5}
	labelCopy="Categories"
	isMultiple
	options={categories}
	on:selected={(e) => {
		console.log(e);
	}}
/>
```
2. Print event from `on:selected` 
3. Event returns only first selected item (I had 3 options selected).
![image](https://user-images.githubusercontent.com/22003852/159759965-804c616e-ab3b-497b-b448-a2f296f093b1.png)

**Expected behavior**

Custom Event will provide enough information to know what options were selected.

**Desktop:**
 - OS: KDE Neon
 - Browser: Brave
 - Version: "agnostic-svelte": "^1.1.10",

**Comment by roblevintennis** [comment]
Yikes! Well that's no good--I have a feeling this is a legitimate bug. 

I'll have a look at it as soon as I can. Seems likely easy to fix hopefully!

**Comment by roblevintennis** [comment]
Bumped version 1.1.12 fixes this @apiotrowski312 and here's the commit:

```shell
[master 5a95c9f8] Fixes #133 multiple select in Svelte
```

I've done tests to ensure both single and multi select work correctly similar to your repro:

```
    <h2>Select large</h2>
    <Select
      uniqueId="sel5"
      name="select5"
      size="large"
      labelCopy="Select the best tennis player of all time"
      defaultOptionLabel="Select your favorite tennis player of all-time"
      options={tennisOptions}
      on:selected={(e) => {
        console.log('Single select: ', e.detail);
      }}
    />
    <h2>Multiple select size 4</h2>
    <div class="mbs12 mbe16">
      <Select
        isMultiple
        multipleSize={4}
        options={tennisOptions}
        on:selected={(e) => {
          console.log('Multi select: ', e.detail);
        }}
        unique-id="sel6"
        name="select6"
        label-copy="Select the best tennis player of all time"
      />
```

And here are the results (stating obvious -- just being thorough! But you need to look at `event.detail` for your selected items):

![image](https://user-images.githubusercontent.com/142403/159801691-4d459e38-77c1-482b-98c6-ad7d2fc592be.png)

___

Note: Fwiw, I've logged a related ticket to check for this in other frameworks (React, Vue, Angular) here:
https://github.com/AgnosticUI/agnosticui/issues/134

Thanks for issue!
